src/server/req_quejob.c

```diff
@@ -1714,7 +1720,10 @@ req_commit_now(struct batch_request *preq,  job *pj)
 	/* set the queue rank attribute */
 	set_jattr_l_slim(pj, JOB_ATR_qrank, time_msec, SET);
 
-	if ((rc = svr_enquejob(pj)) != 0) {
+	if (preq->rq_type == PBS_BATCH_Commit)
```

Comment:
@nithinj I hope you have seen this comment https://github.com/subhasisb/openpbs/blob/f6658057549e1b5c8d4e5963e485ae323c483da1/src/server/req_quejob.c#L1824, Can you please do implicit commit job test with your changes?

Reply:
@hirenvadalia Since implicit commit is part of basic job submission, I hope the regression will cover it.